python/llm/src/ipex_llm/llamaindex/llms/bigdlllm.py

```diff
@@ -171,6 +173,10 @@ class BigdlLLM(CustomLLM):
             "`messages_to_prompt` that does so."
         ),
     )
+    load_low_bit: bool = Field(
```

Comment:
Use an API style similar to langchain
IpexLLM.from_model_id()
IpexLLM.from_low_bit()

Reply:
Fixed.